crates/bevy_winit/src/lib.rs

```diff
@@ -192,6 +193,64 @@ pub fn winit_runner(mut app: App) {
                         });
                     }
                 },
+                WindowEvent::Touch(
```

Comment:
I think its much more legible to compress these into a single match arm, then have a second match inside:

```rust
WindowEvent::Touch(touch) => match touch.phase {
    TouchPhase::Started => {
        let mut touch_finger_input_events =
            app.resources.get_mut::<Events<TouchFingerInput>>().unwrap();
        touch_finger_input_events.send(converters::convert_touch_input(
            ElementState::Pressed,
            &touch,
        ));
    }
    TouchPhase::Ended => {
        let mut touch_finger_input_events =
            app.resources.get_mut::<Events<TouchFingerInput>>().unwrap();
        touch_finger_input_events.send(converters::convert_touch_input(
            ElementState::Released,
            &touch,
        ));
    }
    TouchPhase::Cancelled => {
        let mut touch_finger_input_events =
            app.resources.get_mut::<Events<TouchFingerInput>>().unwrap();
        touch_finger_input_events.send(converters::convert_touch_input(
            ElementState::Released,
            &touch,
        ));
    }
    TouchPhase::Moved => {
        let mut touch_finger_moved_events =
            app.resources.get_mut::<Events<TouchMotion>>().unwrap();
        touch_finger_moved_events.send(TouchMotion {
            finger: Finger(touch.id),
            position: Vec2::new(touch.location.x as f32, touch.location.y as f32),
        });
    }
},
```